### PR TITLE
perf(images): update path of images

### PR DESCRIPTION
**Pull request by braven112**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #59 

## Summary:

Updated all the image references.

## Type of change:

Please delete options that are not relevant.

- [x] Other (Demo image url update)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jordanjones243 — COMMENTED** [review]
When running a build, it shows that there were changes to the files, but none are shown. Otherwise, the changes look good and can be merged once this is investigated.

**Review by jordanjones243 — DISMISSED** [review]

File: demo/api.md
Comment (jason-capsule42) [inline demo/api.md:1]: Shouldn't all the changes here and in index.md be in the docs/partials/ template files?

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: demo/api.md
Comment (jordanjones243) [reply demo/api.md:1]: I don't think so, I am not even sure why it shows so much changed. As far as I know, no content was changed besides the image links.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**